hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcServer.java

```diff
@@ -252,21 +252,21 @@ public class RouterRpcServer extends AbstractService implements ClientProtocol,
   /**
    * Construct a router RPC server.
    *
-   * @param configuration HDFS Configuration.
+   * @param conf HDFS Configuration.
    * @param router A router using this RPC server.
    * @param nnResolver The NN resolver instance to determine active NNs in HA.
-   * @param fileResolver File resolver to resolve file paths to subclusters.
+   * @param fResolver File resolver to resolve file paths to subclusters.
    * @throws IOException If the RPC server could not be created.
    */
-  public RouterRpcServer(Configuration configuration, Router router,
-      ActiveNamenodeResolver nnResolver, FileSubclusterResolver fileResolver)
+  public RouterRpcServer(Configuration conf, Router router,
+      ActiveNamenodeResolver nnResolver, FileSubclusterResolver fResolver)
```

Comment:
Why did you change the name? fileResolver is easier to read.

Reply:
I wanted to remove a line length warning. I'll change it back, "fileResolver" is easier to read.